### PR TITLE
Link against IBL library in iOS projects, update README

### PR DESCRIPTION
**Pull request by bejado**
Filament now depends on `libibl`.

### CODE REVIEW

File: filament/README.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ Copy your platform's Makefile below into a `Makefile` inside the same directory.
 ### Linux
 
 ```
-FILAMENT_LIBS=-lfilament -lbackend -lbluegl -lbluevk -lfilabridge -lfilaflat -lutils -lgeometry -lsmol-v
+FILAMENT_LIBS=-lfilament -lbackend -lbluegl -lbluevk -lfilabridge -lfilaflat -lutils -lgeometry -lsmol-v -libl
```
Comment (romainguy) [inline filament/README.md]: smol-v shouldn't be necessary when Vulkan support is off right?

Comment (bejado) [reply filament/README.md]: It shouldn't, be our releases are compiled with Vulkan support, so it's required here.

Comment (bejado) [reply filament/README.md]: To clarify, only desktop releases should have Vulkan included by default.

Comment (prideout) [reply filament/README.md]: Just to be clear our Android releases are definitely not compiled with Vulkan support but the desktop build is, so this comment is the README is fine as-is.

**Pull request merged**